### DISCUSSION TITLE
Station LOCK feature

### DISCUSSION THREAD

**Discussion by AaronJowitt** [PE5PVB/TEF6686_ESP32#353] [Ideas]
Classic DX tuners with rotary tuning wheels like the Yamaha TX1000 & TX950 have a dedicated front panel 'lock' toggle button, which is useful to prevent an accidental knock of some buttons or the rotary wheel from changing station or its parameters.

A station 'lock' feature would be even more useful added to a portable handheld TEF.

Suggest it would lock at least Freq, Band & Squelch, and possibly Mode (St/Mono), BW, volume.  

Long press of a button to 'unlock'.

**Comment by PE5PVB** [comment]
Yes, but how to activate the lock? All buttons are already dedicated.

**Reply by AaronJowitt** [reply]
Great question. Some possibilities;

1. Long press of two buttons together
2. Long press of rotary button, and move IMS & EQ toggle to settings menu (not sure how often users toggle these...)
3. When rotary is in BW adjust mode, rotary button sets lock
4. When in settings menu, long press of tuning mode button. i.e. two consecutive long presses of same button.
5. When in settings menu, short press of St/mo button or band button.

If none of those (or similar) appeal, then it would still be useful to have lock in the settings menu, preferably first page to get to quicker.

**Comment by PE5PVB** [comment]
Only way is to add a lock as a menu option.
